CSharp/Samples/Microsoft.Bot.Builder.Abstractions.Teams/TeamsActivityProcessor.cs

```diff
@@ -239,13 +246,17 @@ private async Task<InvokeResponse> ProcessTeamsInvokeActivityAsync(ITurnContext
             }
 
             if (teamsContext.IsRequestMessagingExtensionFetchTask())
-            {
-                return await this.invokeActivityHandler.HandleMessagingExtensionFetchTaskAsync(turnContext, teamsContext.GetMessagingExtensionActionData()).ConfigureAwait(false);
+            {   
+                MessagingExtensionAction messagingExtensionActionData = teamsContext.GetMessagingExtensionActionData();
```

Comment:
Do not modify the content of the incoming request. User can choose to strip the html or not. Modifying the model is not right at all.

Reply:
fixed